### PR TITLE
Announce release 20.01

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: src/news/2020-1-galaxy-release/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: 'Galaxy Release 20.01'
+tease: Workflow Reporting, Workflow Executions, Screenreader-friendly navigation
+date: '2019-02-27'
```
Comment (almahmoud) [inline src/news/2020-1-galaxy-release/index.md]: 2020 :)

Comment (mvdbeek) [reply src/news/2020-1-galaxy-release/index.md]: haha, good catch.

Comment (mvdbeek) [reply src/news/2020-1-galaxy-release/index.md]: that would have been really on time though ;)

**Comment by tnabtaf** [comment]
Going in.

**Pull request merged**